src/test/java/uk/gov/hmcts/probate/service/ConfirmationResponseServiceTest.java

```diff
@@ -810,6 +810,33 @@ public void shouldGetNextStepsConfirmationWithNoPayment() {
         assertEquals("0.00", nextStepsValues.get("{{paymentAmount}}"));
     }
 
+    @Test
```

Comment:
There is also a confirmationServiceFeatureTest - does that need updating? I see tests on there for dead execs

Reply:
OK - thanks